src/main/java/com/tictactoe/Game.java

```diff
@@ -4,10 +4,10 @@
     private Rules rules = new RulesFor3x3();
     private Board board;
     private Grid grid;
-    private HumanPlayer player1;
-    private HumanPlayer player2;
+    private Player player1;
+    private Player player2;
 
-    public Game(HumanPlayer player1, HumanPlayer player2, Board board) {
+    public Game(Player player1, Player player2, Board board) {
```

Comment:
I like how you passed the players into the game as arguments. This will make it really easy to test the methods of the game.